.gitignore

```diff
@@ -12,3 +12,4 @@ __pycache__
 cloud-spanner-emulator*
 emulator_main
 gateway_main
+.spanner_emulator/**
```

Comment:
(nit) add `/` at the beginning of the line, so it only matches in this directory.